web/vuenuxt/third_party/webgems/README.md

```diff
@@ -0,0 +1,14 @@
+## Webgems
+
+Original source of this example is from github repository [lostdesign/webgems](https://github.com/lostdesign/webgems).
+This example is Nuxt.js Serverside rendering application listening on port 80.
+In this example we use minification using flag `--include-app-nuxt-static-dir` which includes files required by nuxt.js to work. During minification many files are removed which results in difference of 298,4 MB between big and unoptimized image. 
+
+### How to run and interact
+You can build this image using `make build_fat` command which will output image named slimexamples/server-thirdparty-webgems or docker build . -t {image name}
+To run this unoptimized image you can execute `make run_fat` or `docker run -p 3000:3000 {image name}`.
+
+To interact you can access it using your browser using http://127.0.0.1:1300, if you are running this contianer on another device then you are trying to access it, you need to replace 127.0.0.1 with ip of web address of other device.
```

Comment:
is it supposed to be 3000 or 1300?

Reply:
it was supposed to be 3000 i think i was testing wrong way so i didnt even think about ports, and base of README i just copied so that happens